IDE/ROWLEY-CROSSWORKS-ARM/benchmark_main.c

```diff
@@ -94,7 +93,6 @@ SAMPLE OUTPUT: Freescale K64 running at 96MHz with MMCAU enabled:
 Benchmark Test 0:
 AES      25 kB took 0.019 seconds,    1.285 MB/s
 ARC4     25 kB took 0.033 seconds,    0.740 MB/s
-RABBIT   25 kB took 0.028 seconds,    0.872 MB/s
```

Comment:
Same here: I would leave the Rabbit benchmarks for historical information.
For reference these only got run because if not using configure `NO_RABBIT` had to be set manually to disable.